### DISCUSSION TITLE
Close git controls on iOS

### DISCUSSION THREAD

**Discussion by PetrBNDKT** [Vinzent03/obsidian-git#830] [Problems] [answered]
Hi,
I just set up my repository and installed your plugin to Obsidian on my Mac and my iPhone. I've decided to set my repository on iCloud and used an access token to sync my repository. I've used the same access token on my iPhone and everything seems to worked as expected so far.
But only problem I have on my iPhone when using the plugin is that there is no button to close the source control panel. Every time I open git source control, I can see the diffs, I can push/pull my changes but when I'm done I can't go back to my vault to see my files. At that point I can only force close the app and when I reopen obsidian I'm back in my vault. Is there another way to close the source control tab on mobile? And can you please add close button in future release?
With Regards
Petr B.

![Git source control in iOS](https://github.com/user-attachments/assets/de392cbe-d115-4262-9200-fdfbfabbd43c)

**Comment by Vinzent03** [comment, marked as answer]
The view should be just like the other views from Obsidian itself. If you click on "Source Control" you can switch to other view types. I guess they don't have a close button as well. For me on android the view doesn't fill the whole width though. You might try swiping the view to the left or right.

**Reply by PetrBNDKT** [reply]
Great, that actually helped to solve this. Swiping from left to right does the trick. Actually on my iphone other views have close bottons and doesn't fill the whole screen also, so it's possible to touch the top row which closes the the view as well. So I wasn't expect to swipe that away in this direction. I'm glad that works that way. Thanks for the help.

**Reply by Vinzent03** [reply]
Which theme are you using? This could be the reason.

**Reply by PetrBNDKT** [reply]
I'm using default theme. Haven't tried anything different from it.
